Server/tests/services/jobQueue.test.js

```diff
@@ -30,6 +30,14 @@ class QueueStub {
 	async getJobs() {
 		return this.jobs;
 	}
+
+	async pause() {
+		return true;
+	}
+
+	async obliterate() {
+		return true;
+	}
```

Comment:
Out of curiosity: what do these functions do?

Reply:
> Out of curiosity: what do these functions do?

They are stubs for the actual functions in the JobQueue, we don't actually want to call the methods, we just want to mock them for testing purposes